### ISSUE TITLE
ERROR:ImportError: cannot import name 'text' from 'magnum' 

### ISSUE THREAD

**Issue by WannaSir** [facebookresearch/habitat-sim#2488]
when i run the command:
python examples/viewer.py --scene /path/to/data/scene_datasets/habitat-test-scenes/skokloster-castle.glb
there will be an error:
Traceback (most recent call last):
  File "/data/nvme2/dky/lc/habitat-sim/examples/viewer.py", line 19, in <module>
    from magnum import shaders, text
ImportError: cannot import name 'text' from 'magnum' (/XX/miniconda3/envs/habitat/lib/python3.9/site-packages/magnum/__init__.py)

**Comment by Jwonno** [comment]
I had exactly same errors, did you solve it?

**Comment by zamo24** [comment]
@MuXiangWen I am having this issue too. Have you found a fix yet?

**Comment by fanjing8** [comment]
i use the [solution ](https://github.com/facebookresearch/habitat-sim/issues/2049#issuecomment-1490568606), but i still have this issue.

**Comment by 2927803072** [comment]
> i use the [solution ](https://github.com/facebookresearch/habitat-sim/issues/2049#issuecomment-1490568606), but i still have this issue.

@fanjing8 I met the problem too, do you solve it?

**Comment by fanjing8** [comment]
> > i use the [solution ](https://github.com/facebookresearch/habitat-sim/issues/2049#issuecomment-1490568606), but i still have this issue.
> 
> [@fanjing8](https://github.com/fanjing8) I met the problem too, do you solve it?

No. I tried to build magnum and magnum-bindings from sources, but all solutions failed.
finally I comment some code related to magnum.text in examples/viewer.py

**Comment by 2927803072** [comment]
你好，你有遇到过这个问题吗？
Segmentation fault (core dumped)
------------------&nbsp;原始邮件&nbsp;------------------
发件人:                                                                                                                        "facebookresearch/habitat-sim"                                                                                    ***@***.***&gt;;
发送时间:&nbsp;2025年4月2日(星期三) 上午9:16
***@***.***&gt;;
***@***.******@***.***&gt;;
主题:&nbsp;Re: [facebookresearch/habitat-sim] ERROR:ImportError: cannot import name 'text' from 'magnum'  (Issue #2488)



     
i use the solution , but i still have this issue.
  
@fanjing8 I met the problem too, do you solve it?
  
No. I tried to build magnum and magnum-bindings from sources, but all solutions failed.
 finally I comment some code related to magnum.text in examples/viewer.py

—
Reply to this email directly, view it on GitHub, or unsubscribe.
You are receiving this because you commented.Message ID: ***@***.***&gt;
  fanjing8 left a comment (facebookresearch/habitat-sim#2488)
   
i use the solution , but i still have this issue.
  
@fanjing8 I met the problem too, do you solve it?
  
No. I tried to build magnum and magnum-bindings from sources, but all solutions failed.
 finally I comment some code related to magnum.text in examples/viewer.py
